packages/http/src/validator/validators/body.ts

```diff
@@ -39,8 +63,19 @@ export function deserializeFormBody(
         const deserializer = body[encoding.style];
         const propertySchema = schema.properties?.[property];
 
-        if (propertySchema && typeof propertySchema !== 'boolean')
-          deserialized[property] = deserializer(property, decodedUriParams, propertySchema);
+        if (propertySchema && typeof propertySchema !== 'boolean') {
+          let deserialized_values = encoding.explode !== undefined
+            ? deserializer(property, decodedUriParams, propertySchema, encoding.explode)
+            : deserializer(property, decodedUriParams, propertySchema);
```

Comment:
This is a fix for an existing bug I noticed -- before, user-specified explode settings were never processed. The ternary is because there are a range of default values for explode for the various encodings style functions that I didn't want to touch.